### ISSUE TITLE
Link previews

### ISSUE THREAD

**Issue by the-frey** [envoylabs/whoami-ui#74]
Would be good for preview links to show image if any plus high level meta

**Comment by danwlsn** [comment]
@the-frey by link previews do you mean things like twitter cards? The extra info that pulls through when sharing links on twitter, discord, etc?

**Comment by the-frey** [comment]
Yep, so e.g. on discord at the moment you just get some generic meta. We know people are sharing these quite widely (esp on twitter) so it's worth looking into how hard it is to implement this.
